### PR TITLE
Fixed #25912 --Added binary left/right shift operators to F expressions.

### PR DESCRIPTION
**Pull request by anabelensc**
[django-developers discussion](https://groups.google.com/forum/#!searchin/django-developers/left$20shift%7Csort:relevance/django-developers/KY6N6U_gEdg/ocwjbPo0AQAJ)

### CODE REVIEW

File: docs/releases/1.10.txt [hunk 1]
```diff
@@ -345,6 +345,8 @@ Miscellaneous
 * ``utils.version.get_version()`` returns :pep:`440` compliant release
   candidate versions (e.g. '1.10rc1' instead of '1.10c1').
 
+* The :class:`F expressions <django.db.models.F>` supported bitwise shift operators by ``>>`` and ``<<``.These operators are now available using ``.bitleftshift()`` and ``.bitrightshift()``.
+
```
Comment (charettes) [inline docs/releases/1.10.txt]: Try to keep docs line length below 80 characters.

**Comment by charettes** [comment]
Could you rebase on top of the latest master, it looks like there's some merge conflicts.

**Comment by anabelensc** [comment]
Done.

**Comment by auvipy** [comment]
@anabelensc still merge conflict

**Comment by auvipy** [comment]
@anabelensc could you fix the doc build errors and merge conflict?

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by timgraham** [comment]
Could you create a [django-developers discussion](https://groups.google.com/forum/#!forum/django-developers) as per https://code.djangoproject.com/ticket/25912#comment:10 to verify this is the desired implementation?

**Comment by anabelensc** [comment]
Done!you can check the thread I have created django-developers discussion, thank you

**Comment by adamchainz** [comment]
It took me some time to find the discussion from herer, here's the link to save anyone time: https://groups.google.com/forum/#!searchin/django-developers/left$20shift%7Csort:relevance/django-developers/KY6N6U_gEdg/ocwjbPo0AQAJ

I think adding this would be good to add for completion. I'd say it's slightly less surprising to implement the bit shifting with maths on Oracle rather than raising `NotImplementedError`, given that for other database functions such work arounds tend to get used.

**Comment by timgraham** [comment]
All right, absent other feedback, I guess we'll proceed with this. Patch needs to be targeted at Django 1.11 and updated per Adam's comments.

**Comment by atombrella** [comment]
https://docs.oracle.com/cd/E22583_01/DR/help/Bit_Binary_Functions.html#DALc02a_2726006080_257082 Oracle implements this with `bitshift`.

**Comment by timgraham** [comment]
I copied Oracle's implementation from BITAND but get `ORA-00904: "BITSHIFT": invalid identifier`. Not sure what could be wrong. \cc @felixxm

**Comment by timgraham** [comment]
The linked documentation seems to be for something called "Document Automation Language (DAL)". I guess this is different than what the Oracle database supports.

**Comment by timgraham** [comment]
Updated in https://github.com/django/django/pull/7770.

**Comment by felixxm** [comment]
To be honest I have never heard about DAL, but I think that we can not use it in `SQL` or `PL/SQL`.

**Pull request closed without merging**